### ISSUE TITLE
[Question]  lobe-i18n cli run failed?

### ISSUE THREAD

**Issue by xieyezi** [lobehub/lobe-cli-toolbox#67]
### 🧐 问题描述 | Proposed Solution

First of all, thank you very much for providing this tool！
 I'm having some problems now.

This my script: 

```js
...
  execSync(`lobe-i18n md --config ${componentTranslateConfigPath}`, {
    cwd: process.cwd(),
    stdio: "inherit",
    env: {
      OPENAI_PROXY_URL: "xxxx",
      OPENAI_API_KEY: "xxxx"
    }
  });
...
```

and this is my `.i18nrc.js`: 

```js
const path = require("path");
const { defineConfig } = require("@lobehub/i18n-cli");

module.exports = defineConfig({
  markdown: {
    entry: [path.resolve(__dirname, "./index.md")],
    entryLocale: "en-US",
    entryExtension: ".md",
    outputLocales: ["zh-CN"],
    outputExtensions: (locale, { getDefaultExtension }) => {
      //if (locale === "en-US") return ".md";
      return getDefaultExtension(locale);
    }
  }
});
```

All the config load normally, but it's failed:

```
ℹ start translate portal doc...                                                17:17:02
o Lobe I18N is analyzing your markdown... 🤯🌏🔍                          17:17:05
o Running in 📄 1 Markdown, and translating to zh-CN locales..            17:17:05
i 📄 To zh-CN: src\portal\index.md                              17:17:05  
i Current model setting: gpt-3.5-turbo (temperature: 0) }                  17:17:05  
──── 📝 E:\project\incall-fuxi\packages\fuxi\src\portal\index.zh-CN.md 
- from en-US to zh-CN [Tokens: 1619]
⠧  0% [0/1 chunks] 

 ERROR  Translate failed, TypeError: Cannot read properties of undefined (reading 'message')        17:17:07  
```

**Comment by canisminor1990** [comment]
@xieyezi  可能需要你这边排除下问题
- 直接用命令行执行 cli 是否成功
- 试试在 index.md 中只输入简单语句看看是否正确执行翻译
- 如果上述可以执行，尝试吧 splitToken 改小

**Comment by xieyezi** [comment]
> @xieyezi 可能需要你这边排除下问题
> 
> * 直接用命令行执行 cli 是否成功
> * 试试在 index.md 中只输入简单语句看看是否正确执行翻译
> * 如果上述可以执行，尝试吧 splitToken 改小

你好，请问直接在命令行执行，如何传入` OPENAI_API_KEY `和 `OPENAI_PROXY_URL` 两个环境变量呢

**Comment by xieyezi** [comment]
我直接使用 cli方式进行调用：

```bash
export OPENAI_PROXY_URL=xxx
export OPENAI_API_KEY=xxx
lobe-i18n md -c ./.i18nrc.js
```
这是我的`markdown`文件:

```
// index.md

## Features

- Multi portals handling.
- Multi portal hosts handling.
- Allow override functionality.
- Compatible with React Native Web.
- Compatible with Expo, check out the project Expo Snack.
- Written in TypeScript.

```

但是仍然无法调用，提示如下错误：

```shell
$ lobe-i18n md -c ./.i18nrc.js
◐ Lobe I18N is analyzing your markdown... 🤯🌏🔍                     15:21:01
◐ Running in 📄 1 Markdown, and translating to zh-CN locales..               15:21:01            
ℹ 📄 To zh-CN: index.md             15:21:01   
ℹ Current model setting: gpt-3.5-turbo (temperature: 0) }        15:21:01   
─── 📝 E:\project\incall-fuxi\packages\fuxi\src\portal\index.zh-CN.md 
- from en-US to zh-CN [Tokens: 200]
⠋  0% [0/1 chunks] ░░░░░░░░░░░

 ERROR  Translate failed, TypeError: Cannot read properties of undefined (reading 'message')       15:21:03
```

**Comment by canisminor1990** [comment]
curl 试试 ？ curl {{OPENAI_PROXY_URL}}/engines -H "Authorization: Bearer {{OPENAI_API_KEY}}"

**Comment by xieyezi** [comment]
Thank you for your hard work and dedication in creating such great tools. I encountered a similar issue with lobeChat before, where responses failed. After numerous attempts, I discovered that by simply appending `/v1` to my proxy address (https://proxy-url/v1), all features, including `lobeChat` and `lobeI18n`, started working properly. 

Thank you once again for your strenuous efforts, it's so cool!
